sdk/core/Azure.Core.TestFramework/README.md

```diff
@@ -79,6 +79,24 @@ public class AppConfigurationTestEnvironment : TestEnvironment
 
 **NOTE:** Make sure that variables containing secret values are not recorded or are sanitized.
 
+To sanitize variables use the `options` parameter of `GetRecordedVariable`:
+
+``` C#
```

Comment:
Nit: maybe use the sample snippet feature to make sure this is in sync? The internal-ish stuff has gotten refactored in the past, so this would keep it working.